backend/WP/Gutenberg/js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@wordpress/babel-plugin-import-jsx-pragma": "^2.2.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "parcel-bundler": "^1.12.3"
+  },
+  "scripts": {
+    "start": "parcel gutenberg.js",
+    "build": "parcel build gutenberg.js"
+  },
+  "dependencies": {
+    "@wordpress/components": "^8.0.0",
+    "@wordpress/compose": "^3.4.0",
+    "@wordpress/element": "^2.5.0",
```

Comment:
import { Component } from '@wordpress/element'
const { Component } = wp.element

If you are going to use the second syntax, then it's not necessary to install the @wordpress/element package.